linchpin/InventoryFilters/DuffyInventory.py

```diff
@@ -28,25 +28,30 @@ def get_host_data(self, topo, cfgs):
         :param cfgs:
             map of config options from PinFile
         """
-
         host_data = {}
+        if res['resource_type'] != 'duffy_res':
```

Comment:
are we still using resource_type ??? 
I thought it was changed to role long back !!
@14rcole

Reply:
I changed `resources` in linchpin.latest to an array of things that had been provisioned rather than a dict with an array for each provider.  Because of this, I had to add a key to denote what resource type had just been provisioned and I chose `resource_type`.  I can change easily it to something else if you'd like